src/gsAssembler/gsQuadRule.h

```diff
@@ -69,9 +57,21 @@ class gsQuadRule
      *
      */
     virtual void setNodes( gsVector<index_t> const & numNodes,
-                           unsigned digits = std::numeric_limits<T>::digits10 )
+                           unsigned digits = REAL_DIG )
```

Comment:
we might prfer c++ over c

Reply:
This is defined in gsMath.h and is equal to std::numeric_limits<T>::digits10